src/Infrastructure.Dapper/Tools/Helpers/SendHelpers.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System.Data;
+using Bit.Core.Tools.Entities;
+
+namespace Bit.Infrastructure.Dapper.Tools.Helpers;
+
+/// <summary>
+/// Dapper helper methods for Sends
+/// </summary>
+public static class SendHelpers
+{
+    /// <summary>
+    /// Converts an IEnumerable of Sends to a DataTable
+    /// Note: Contains a hardcoded list of properties and must be updated with model
+    /// </summary>
+    /// <param name="sends">List of sends</param>
+    /// <returns><see cref="DataTable"/></returns>
```

Comment:
⛏️ I'd be more descriptive. "A data table matching the schema of `dbo.Send` containing one row mapped from the items in `sends`" (with the appropriate refs ofc.)

Reply:
Can't figure out how to reference the file, but changed

Reply:
I don't think there's a way to ref the file, unfortunately. :(